config/settings/review.yml

```diff
@@ -16,3 +16,5 @@ authentication:
   secret: secret
   mode: persona
 
+features:
+  teacher_degree_apprenticeship: true
```

Comment:
This will be removed before merging, just want to be able to test this in review app